### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Dreller** [notrinos/NotrinosERP#9] [General]
Hi, how may I create additional translation files ?  I would like to translate it in French (fr-CA) but I only find a .mo file (need the .po file to be able to edit it).  Thanks!

**Comment by Dreller** [comment]
Just found the answer :)

If the language file you require doesn't exist on the Sourceforge server you can create your own .po and .mo files using [[http://www.poedit.net/ | Poedit]].  
Simply use Poedit to alter an existing .po language file where you wish to change it and save to your new language folder. Then update the language as above. In Windows, the program '''C:\Program Files\Poedit\bin\msgunfmt.exe''' can be used to revert a compiled '''.mo''' file back into a '''.po''' file to be edited and then re-compiled using '''C:\Program Files\Poedit\bin\msgfmt.exe'''. The command '''msgunfmt.exe -o en_US.po en_US.mo''' in a DOS box does the reversion.

**Reply by notrinos** [reply]
Hello @Dreller 
There is already a French translation (fr_FR) in NotrinosERP
Go to **Setup -> Install/Update Languages** then download the needed translation
After download, open your lang directory you will get both po and mo files here
Would you like to share the new fr_CA translation if it is ready?

**Reply by Dreller** [reply]
Thank you for this info!  I will certainly share the fr_CA once it's ready!
